app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admin::UsersController < Admin::BaseController
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    user = User.new(permitted_params)
+    user.verified_at = Time.current
```

Comment:
let's add a method called set_verified_at, that does this

Reply:
Done

Reply:
Created a private method `set_verified_at` for setting the field